packages/vue-components/src/questions/QOption.vue

```diff
@@ -214,4 +314,26 @@ export default {
         margin: 0.2rem 0 0 0;
         align-items: center;
     }
+
+    /* for multiBlanks question type */
+    input.form-control {
+        height: auto;
+        min-height: 20px;
+        margin-bottom: 0;
+        width: 50%;
+    }
+
+    input.form-control:disabled,
+    .multiBlanks-keywords {
+        margin-bottom: 0.5rem;
+    }
+
+    .multiBlanks-div {
```

Comment:
We're following [this](https://google.github.io/styleguide/htmlcssguide.html#ID_and_Class_Name_Delimiters) ..which dosen't say how to handle compound words =(.. but [this](https://developers.google.com/style/word-list#compounds) does.

i.e. would be `multiblanks-div`.

Or perhaps you could get rid of `multi` in the code references too, per the earlier suggestion.

A clearer suffix might be `multiblanks-form-control` / `blanks-form-control` as well so the reader immediately knows this overrides `form-control` for the case of blanks questions.